src/Revit/RevitHyparTools/Create.Wall.cs

```diff
@@ -0,0 +1,41 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Elements.Geometry;
+using GeometryEx;
+using ElemGeom = Elements.Geometry;
+
+using Revit = Autodesk.Revit.DB;
+
+namespace RevitHyparTools
+{
+    public static partial class Create
```

Comment:
Remind me why this is `partial`.

Reply:
the pattern I have is that there is one Create class spread across multiple files depending on what is being created.  It is more obvious in subsequent work with Floor and columns following the same pattern